front/lib/api/assistant/global_agents.ts

```diff
@@ -337,13 +338,9 @@ function _getO1PreviewGlobalAgent({
       modelId: O1_PREVIEW_MODEL_CONFIG.modelId,
       temperature: 1, // 1 is forced for O1
     },
-    actions: [
-      _getDefaultSearchActionForGlobalAgent({
```

Comment:
@overmode I believe you forgot to remove this: https://github.com/dust-tt/dust/pull/8637/files#diff-abbc0e8a2e1d1b7961d9c038fc4cb300b11f5906e0405ccbe421f43b01f63c33R334

Which means we now have an inconstent prompt affecting all Anthropic based assistants. Given the not so great impact creating a PR to fix.